buildSrc/src/test/java/org/springframework/boot/build/mavenplugin/PluginXmlParserTests.java

```diff
@@ -16,25 +16,49 @@
 
 package org.springframework.boot.build.mavenplugin;
 
-import java.io.File;
-
 import org.junit.jupiter.api.Test;
-
 import org.springframework.boot.build.mavenplugin.PluginXmlParser.Plugin;
 
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
 /**
  * Tests for {@link PluginXmlParser}.
  *
  * @author Andy Wilkinson
+ * @author Mike Smithson
  */
 public class PluginXmlParserTests {
 
 	private final PluginXmlParser parser = new PluginXmlParser();
 
 	@Test
-	void dunno() {
+	void parsing_a_plugin_xml_file_returns_the_gav_for_plugin() {
 		Plugin plugin = this.parser.parse(new File("src/test/resources/plugin.xml"));
-		System.out.println(plugin);
+
+		assertThat(plugin.getGroupId()).isEqualTo("org.springframework.boot");
+
+		assertThat(plugin.getArtifactId()).isEqualTo("spring-boot-maven-plugin");
+
+		assertThat(plugin.getVersion()).isEqualTo("2.2.0.GRADLE-SNAPSHOT");
+
+		assertThat(plugin.getGoalPrefix()).isEqualTo("spring-boot");
+
+		List<String> actualMojoGoals = plugin.getMojos().stream().map(PluginXmlParser.Mojo::getGoal).collect(Collectors.toList());
+		List<String> expectedMojoGoals = Arrays.asList("build-info", "help", "repackage", "run", "start", "stop");
```

Comment:
I would move that to the assertion rather than creating a variable for that.

Reply:
Inlined as of commit 976fc62